bitcraze_crazyflie_2/cf2.xml

```diff
@@ -0,0 +1,119 @@
+    <mesh file="cf2_collision_30.obj"/>
+    <mesh file="cf2_collision_31.obj"/>
+  </asset>
+  <worldbody>
+    <body name="cf2" pos="0 0 0.1">
+      <inertial pos="0 0 0" mass="0.027" diaginertia="2.3951e-5 2.3951e-5 3.2347e-5"/>
+      <joint name="root" type="free" damping="0" armature="0" pos="0 0 0" limited="false"/>
+      <geom mesh="cf2_0" material="Material15_001" class="visual"/>
+      <geom mesh="cf2_1" material="Material14-green_medium_gloss_plastic_001" class="visual"/>
+      <geom mesh="cf2_2" material="Material12-polished_gold_001" class="visual"/>
+      <geom mesh="cf2_3" material="Material10-polished_plastic1_001" class="visual"/>
+      <geom mesh="cf2_4" material="Material8-burnished_chrome_001" class="visual"/>
+      <geom mesh="cf2_5" material="Material6-black_medium_gloss_plastic_001" class="visual"/>
+      <geom mesh="cf2_6" material="Material3-polypropylene2_001" class="visual"/>
+      <geom mesh="cf2_collision_0" class="collision"/>
+      <geom mesh="cf2_collision_1" class="collision"/>
+      <geom mesh="cf2_collision_2" class="collision"/>
+      <geom mesh="cf2_collision_3" class="collision"/>
+      <geom mesh="cf2_collision_4" class="collision"/>
+      <geom mesh="cf2_collision_5" class="collision"/>
+      <geom mesh="cf2_collision_6" class="collision"/>
+      <geom mesh="cf2_collision_7" class="collision"/>
+      <geom mesh="cf2_collision_8" class="collision"/>
+      <geom mesh="cf2_collision_9" class="collision"/>
+      <geom mesh="cf2_collision_10" class="collision"/>
+      <geom mesh="cf2_collision_11" class="collision"/>
+      <geom mesh="cf2_collision_12" class="collision"/>
+      <geom mesh="cf2_collision_13" class="collision"/>
+      <geom mesh="cf2_collision_14" class="collision"/>
+      <geom mesh="cf2_collision_15" class="collision"/>
+      <geom mesh="cf2_collision_16" class="collision"/>
+      <geom mesh="cf2_collision_17" class="collision"/>
+      <geom mesh="cf2_collision_18" class="collision"/>
+      <geom mesh="cf2_collision_19" class="collision"/>
+      <geom mesh="cf2_collision_20" class="collision"/>
+      <geom mesh="cf2_collision_21" class="collision"/>
+      <geom mesh="cf2_collision_22" class="collision"/>
+      <geom mesh="cf2_collision_23" class="collision"/>
+      <geom mesh="cf2_collision_24" class="collision"/>
+      <geom mesh="cf2_collision_25" class="collision"/>
+      <geom mesh="cf2_collision_26" class="collision"/>
+      <geom mesh="cf2_collision_27" class="collision"/>
+      <geom mesh="cf2_collision_28" class="collision"/>
+      <geom mesh="cf2_collision_29" class="collision"/>
+      <geom mesh="cf2_collision_30" class="collision"/>
+      <geom mesh="cf2_collision_31" class="collision"/>
+
+      <site name="actuation" type="sphere" size="0.001 0.001 0.001" rgba="1 0 0 1" pos="0 0 0"/>
+    </body>
+  </worldbody>
+  <actuator>
+    <motor ctrllimited="true" ctrlrange="0 0.35" gear="0 0 1 0 0 0" site="actuation" name="body_thrust"/>
```

Comment:
Add `autolimits=true` in the compiler and then remove `ctrllimited` from all the actuators.